src/style.css

```diff
@@ -0,0 +1,149 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+}
+
+.card__product {
+  padding-top: 32px;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__image {
+  display: inline-block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.card__id {
+  margin-bottom: 16px;
+}
+
+.card__info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 0 4px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  /* margin-top: 20px; */
+  margin-bottom: 16px;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  background-image: url(images/star.svg);
+}
+
+.card__review-counts {
+  display: flex;
+  align-items: flex-end;
+}
+
+.card__price-sticker {
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  justify-content: space-between;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  size: 16px;
```

Comment:
Change this styles according to mockup
![Screenshot 2022-07-19 at 11 08 45](https://user-images.githubusercontent.com/17098736/179700446-a7856b5c-ed89-42f3-89b1-f1cec2fa33ee.png)
.